source/features/index.tsx

```diff
@@ -123,7 +129,7 @@ const globalReady: Promise<RGHOptions> = new Promise(async resolve => {
 
 	if (select.exists('body.logged-out')) {
 		console.warn('Refined GitHub is only expected to work when you’re logged in to GitHub. Errors will not be shown.');
-		features.error = () => {/* No logging */};
+		features.log.error = () => {/* No logging */};
 		logError = () => {/* No logging */};
```

Comment:
If we use `features.log.error` exclusively, we no longer need to override `logError` too
```suggestion
		features.log.error = () => {/* No logging */};
```

Reply:
I couldn't commit your suggestion because it shows "This suggestion is invalid because no changes were made to the code", so I did it manually. Thank you.